virttest/data_dir.py

```diff
@@ -215,6 +237,11 @@ def get_tmp_dir(public=True):
         tmp_dir_st = os.stat(tmp_dir)
         os.chmod(tmp_dir, tmp_dir_st.st_mode | stat.S_IXUSR |
                  stat.S_IXGRP | stat.S_IXOTH | stat.S_IRGRP | stat.S_IROTH)
+    if "tmpfs" == get_fs_type(tmp_dir):
+        logging.warning("tmp_dir is type tmpfs. This might lead to issues"
+                        " e.g. when requiring direct access. Please, consider"
+                        " using data_dir or configuring another path in vt.common")
```

Comment:
What if we simply use the tmp_dir for storing a few files with faster IO? Should this warning always be shown? Or is it planned to be temporary?